pom.xml

```diff
@@ -620,13 +621,30 @@
             <artifactId>revapi-java</artifactId>
             <version>${revapi-java.version}</version>
           </dependency>
+          <dependency>
+            <groupId>org.revapi</groupId>
+            <artifactId>revapi-reporter-json</artifactId>
+            <version>${revapi-reporter-json-version}</version>
+          </dependency>
         </dependencies>
         <configuration>
+          <failBuildOnProblemsFound>false</failBuildOnProblemsFound>
```

Comment:
Would it still output the report if we set this value to `true`? Since the backend currently does not read RevApi reports it would make sense to postpone the removal until the backend is active.

Reply:
Yes the report still gets generated when it fails. When this is true and revApi finds a change which breaks the semantic version the build will fail (verify, install, deploy). When it is false the operation succeeds regardless of the result of revApi check.

Reply:
Ok, then set this property to `true` and we postpone that part of the change.